src/util/helpers.ts

```diff
@@ -54,11 +54,16 @@ export function getWebSocketClass(
 /**
  * Given connection metadata, creates a websocket connection string
  */
-export function getConnectionStr(connectionMetadata: GovalMetadata, isPolling: boolean): string {
+export function getConnectionStr(
+  connectionMetadata: GovalMetadata,
+  isPolling: boolean,
+  pollingHost: string | null,
```

Comment:
```suggestion
  pollingHost?: string,
```

for type congruence with the above suggestion.